osu.Game/Overlays/OSD/Toast.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.OSD
+{
+    public abstract class Toast : Container
+    {
+        private readonly Container content;
+        protected override Container<Drawable> Content => content;
+
+        protected readonly OsuSpriteText ValueText;
+
+        protected Toast(string description, string value, string keybinding)
+        {
+            Anchor = Anchor.Centre;
+            Origin = Anchor.Centre;
+            Width = 240;
```

Comment:
broke width on long text

![image](https://user-images.githubusercontent.com/191335/62838498-13944800-bcb8-11e9-9fc1-45961970d8fd.png)

please write tests for edge cases like this.

Reply:
I guess correct behaviour for this is having the toast autosizing on X axis